File: src/main/Java/boj_step1/boj11382.java
import java.io.*;
import java.util.StringTokenizer;

public class boj11382 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        long a = Long.parseLong(tokenizer.nextToken());
        long b = Long.parseLong(tokenizer.nextToken());
        long c = Long.parseLong(tokenizer.nextToken());
        writer.append(String.valueOf(a + b + c));
        writer.close();
    }
}

File: src/main/kotlin/others/boj11382.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val (a, b, c) = r.readLine().split(" ").map { it.toBigInteger() }
    w.write((a + b + c).toString())
    w.close()
}